kubeadm/hack/publish-flannel.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/Masterminds/semver"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+)
+
+const (
+	supportedVersions = ">= 0.12.0"
+)
+
+func dockerBuild(v *semver.Version, dockerfilePath string) error {
+	cmd := exec.Command("docker", "build", "--pull",
+		fmt.Sprintf("--build-arg=flannelVersion=%s", v.String()),
+		fmt.Sprintf("--tag=sigwindowstools/flannel:%s", v.String()),
+		dockerfilePath,
+	)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	return cmd.Run()
+}
+
+func dockerPush() error {
+	cmd := exec.Command("docker", "push", "sigwindowstools/flannel")
```

Comment:
Just noting that with some of the inbound dockerhub limitations we may have to migrate these to someplace else.

Reply:
Noted.
@ddebroy was filing some exceptions to the inbound dockerhub limitations for the registries most of the e2e test images are stored (mainly e2eteam) since these are open source projects.
We should see if we can get an exception for sigwindowstools registry as well.

Reply:
This code similar to publish-kubeproxy.go